ledger-service/metrics/src/main/scala/com/digitalasset/http/metrics/HttpJsonApiMetrics.scala

```diff
@@ -18,12 +18,16 @@ object HttpJsonApiMetrics {
       new DropwizardMetricsFactory(new MetricRegistry),
       new OpenTelemetryFactory(GlobalOpenTelemetry.getMeter("test")),
     )
+
+  final val ComponentName = "json_api"
```

Comment:
What's the naming convention for components? E.g., do we use dashes or underscores for separating words?
I'm asking because this was named as `json-api` before.

Reply:
I changed it following a comment by @nicu-da 
https://github.com/digital-asset/daml/pull/16052/commits/d140fa5a711985840948b24d6949339d697283a1#r1073699317